### ISSUE TITLE
Retrieving WaveAnalyticAssetCollection from Org Browser Throws Error: Missing metadata type definition in registry for id 'waveanalyticassetcollection'

### ISSUE THREAD

**Issue by nickytorstensson** [forcedotcom/salesforcedx-vscode#5855]
### Summary
In VS Code, using the Salesforce extensions, specifically the Org Browser, I am trying to retrieve the "WaveAnalyticAssetCollection", which seems to be recently introduced (a month ago, in August 2024). I am trying to retrieve this against a normal developer sandbox org, created from a production environment. 

However, it fails with error `Missing metadata type definition in registry for id 'waveanalyticassetcollection'.`

I don't expect to provide a repository, when it is the available Org Browser, from the VS Code extension, that provides this functionality.

### Steps To Reproduce

1. Open VS Code
2. Once all extensions have loaded, click the Salesforce cloud icon to open the Org Browser
3. Scroll to the bottom and click to retrieve the "WaveAnalyticAssetCollection"

### Expected result
Expected that the collection (specifically from screenshot that the "My_Dashboards" is retrieved

### Actual result
Error thrown:  `Missing metadata type definition in registry for id 'waveanalyticassetcollection'.`

### Additional information
![image](https://github.com/user-attachments/assets/63eea227-361c-46bb-92d3-7bd2317ef7fb)

### System Information
I have ensured that latest CLI version and VS Code version is used
- Version: 1.93.1 (user setup)
- Commit: 38c31bc77e0dd6ae88a4e9cc93428cc27a56ba40
- Date: 2024-09-11T17:20:05.685Z
- Electron: 30.4.0
- ElectronBuildId: 10073054
- Chromium: 124.0.6367.243
- Node.js: 20.15.1
- V8: 12.4.254.20-electron.0
- OS: Windows_NT x64 10.0.22621

Which shell or terminal are you using? (bash, zsh, powershell 7, cmd.exe, etc)
- Not sure. I am on a Windows 11 machine, and just running the Terminal

Paste the **full** output of the `sf version --verbose --json` command below
```
{
  "architecture": "win32-x64",
  "cliVersion": "@salesforce/cli/2.58.7",
  "nodeVersion": "node-v20.17.0",
  "osVersion": "Windows_NT 10.0.22621",
  "rootPath": "C:\\Users\\ntorstensson\\AppData\\Local\\sf\\client\\2.58.7-ed27925",
  "shell": "powershell",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 3.2.2 (core)",
    "@oclif/plugin-commands 4.0.13 (core)",
    "@oclif/plugin-help 6.2.10 (core)",
    "@oclif/plugin-not-found 3.2.18 (core)",
    "@oclif/plugin-plugins 5.4.6 (core)",
    "@oclif/plugin-search 1.2.7 (core)",
    "@oclif/plugin-update 4.5.5 (core)",
    "@oclif/plugin-version 2.2.11 (core)",
    "@oclif/plugin-warn-if-update-available 3.1.13 (core)",
    "@oclif/plugin-which 3.2.12 (core)",
    "@salesforce/cli 2.58.7 (core)",
    "apex 3.4.8 (core)",
    "api 1.2.1 (core)",
    "auth 3.6.54 (core)",
    "data 3.6.5 (core)",
    "deploy-retrieve 3.12.3 (core)",
    "info 3.4.3 (core)",
    "limits 3.3.29 (core)",
    "marketplace 1.2.25 (core)",
    "org 4.5.7 (core)",
    "packaging 2.8.2 (core)",
    "schema 3.3.26 (core)",
    "settings 2.3.16 (core)",
    "sobject 1.4.33 (core)",
    "source 3.5.18 (core)",
    "telemetry 3.6.10 (core)",
    "templates 56.3.17 (core)",
    "trust 3.7.27 (core)",
    "user 3.5.29 (core)"
  ]
}
```

**Comment by diyer** [comment]
Hi @nickytorstensson this will be in our next week's release thanks to @CristiCanizales

**Comment by daphne-sfdc** [comment]
We have published this week's release v61.15.0 of the Salesforce Extensions for VSCode, which contains the fix for this bug.

**Comment by nickytorstensson** [comment]
Confirmed working.
